parts/k8s/kubernetesmasterresourcesvmss.t

```diff
@@ -186,22 +186,29 @@
 },
 {{end}}
 {
-  "apiVersion": "[variables('apiVersionDefault')]",
+  "apiVersion": "2017-08-01",
```

Comment:
Any reason for using `2017-08-01` and not a more recent version like `2018-04-01` that we already use for accelerated networking? 

In any case, we have a backlog item to rationalize the default version https://github.com/Azure/acs-engine/issues/3835 so the version will be bumped soon. If the version needs to be `2017-08-01` and is reused in multiple places I would recommend creating a variable for it.